stan/math/prim/mat/prob/ordered_probit_lpmf.hpp

```diff
@@ -0,0 +1,223 @@
+#ifndef STAN_MATH_PRIM_MAT_PROB_ORDERED_PROBIT_LPMF_HPP
+#define STAN_MATH_PRIM_MAT_PROB_ORDERED_PROBIT_LPMF_HPP
+
+#include <stan/math/prim/scal/fun/Phi.hpp>
+#include <stan/math/prim/scal/fun/log1m.hpp>
+#include <stan/math/prim/scal/err/check_bounded.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
+#include <stan/math/prim/scal/err/check_greater.hpp>
+#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
+#include <stan/math/prim/arr/err/check_ordered.hpp>
+#include <stan/math/prim/scal/meta/return_type.hpp>
+#include <Eigen/StdVector>
```

Comment:
I've never seen this include before. Why are you including it (as in what functionality is it bringing in that you're using)?  You don't need to doc, I just want to know.

Reply:
Oops, this was a leftover from earlier testing. Eigen can have [troubles with STL containers and fixed-size matrices/vectors](https://eigen.tuxfamily.org/dox/group__TopicStlContainers.html), so they recommend including this. Not necessary for the Dynamic sizes though